### PR TITLE
Uses curl-sh for top-level quick-start

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Thanks to @abesto, here's how to get the latest zipkin jar

```bash
$ curl -sSL https://zipkin.io/quickstart.sh | bash -s
```

And here's how to get a module, like our kafka10+ addon
```bash
$ curl -sSL https://zipkin.io/quickstart.sh | bash -s io.zipkin.java:zipkin-autoconfigure-collector-kafka10:LATEST:module kafka10.jar
```

See https://github.com/openzipkin/openzipkin.github.io/pull/90

### CODE REVIEW

**Comment by abesto** [comment]
FWIW, LGTM :)

(Need more acronyms?)

File: zipkin-autoconfigure/collector-kafka10/README.md [hunk 1]
```diff
@@ -10,48 +10,48 @@ variable or `zipkin.collector.kafka.bootstrap-servers` property.
 JRE 8 is required to run Zipkin server. Note: The Kafka 0.10+ collector and this auto-configure
 module are compatible with Java 7 and later when used independent of Zipkin server.
 
-Fetch the latest released 
+Fetch the latest released
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
```
Comment (shakuzen) [inline zipkin-autoconfigure/collector-kafka10/README.md:14]: Unfortunately this link does not work due to the ongoing Maven central certificate problem. I'm not sure what's best for us to do about it, though.

Comment (codefromthecrypt) [reply zipkin-autoconfigure/collector-kafka10/README.md:14]: indeed see https://github.com/openzipkin/zipkin/issues/1804 and specifically https://issues.sonatype.org/browse/MVNCENTRAL-2870 which covers the root problem (those watching from home)

**Comment by jorgheymans** [comment]
this doesn't work on my machine (it was working yesterday though). Looking at the debug output it seems that the URL to download the jar from bintray contains some backslashes ```maven/io\/zipkin\/java``` and this is why curl downloads nothing - at least on my machine.

```
+ echo 2.4.0
+ grep -E '^[0-9]+\.[0-9]+\.[0-9]+$'
+ echo 'Downloading io.zipkin.java:zipkin-server:2.4.0:exec to zipkin.jar...'
Downloading io.zipkin.java:zipkin-server:2.4.0:exec to zipkin.jar...
+ local 'url=https://dl.bintray.com/openzipkin/maven/io\/zipkin\/java/zipkin-server/2.4.0/zipkin-server-2.4.0-exec.jar'
+ echo 'https://dl.bintray.com/openzipkin/maven/io\/zipkin\/java/zipkin-server/2.4.0/zipkin-server-2.4.0-exec.jar -> https://dl.bintray.com/openzipkin/maven/io\/zipkin\/java/zipkin-server/2.4.0/zipkin-server-2.4.0-exec.jar'
https://dl.bintray.com/openzipkin/maven/io\/zipkin\/java/zipkin-server/2.4.0/zipkin-server-2.4.0-exec.jar -> https://dl.bintray.com/openzipkin/maven/io\/zipkin\/java/zipkin-server/2.4.0/zipkin-server-2.4.0-exec.jar
+ curl -L -o zipkin.jar 'https://dl.bintray.com/openzipkin/maven/io\/zipkin\/java/zipkin-server/2.4.0/zipkin-server-2.4.0-exec.jar'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0    33    0    33    0     0    123      0 --:--:-- --:--:-- --:--:--   146

```

**Comment by abesto** [comment]
Thanks for reporting! At first glance it smells like a bash version issue.
I'll investigate and fix up the script later today.

**Comment by abesto** [comment]
@jorgheymans I've uploaded a version that works on the machine where I managed to reproduce the issue (https://github.com/openzipkin/openzipkin.github.io/commit/d114d9a24ddda8526e5690127eb2215b3504b1ff). It'd be great if you could re-run the same command you used originally and let me know the results.

**Comment by codefromthecrypt** [comment]
gonna merge this one.. true that our "download" link is not great as that still has a problem. We should merge this anyway as the quick start is certainly busted and this fixes that.

File: zipkin-autoconfigure/collector-kafka10/README.md (hunk 1 above)
Comment (codefromthecrypt) [reply zipkin-autoconfigure/collector-kafka10/README.md:14]: https://github.com/openzipkin/zipkin/issues/1836

**Pull request merged**